app/styles.source.css

```diff
@@ -333,7 +333,7 @@ html.dark {
   }
 
   & pre code {
-    display: inline;
+    display: grid;
```

Comment:
thanks!

`inline-block` should also suffice

```suggestion
    display: inline-block;
```